packages/block-editor/src/components/rich-text/use-input-rules.js

```diff
@@ -142,7 +142,7 @@ export function useInputRules( props ) {
 				return;
 			}
 
-			if ( __unstableAllowPrefixTransformations && inputRule ) {
+			if ( __unstableAllowPrefixTransformations ) {
 				if ( inputRule() ) return;
 			}
```

Comment:
We can actually remove the nested `if` and keep it a single one:

```suggestion
			if ( __unstableAllowPrefixTransformations && inputRule() ) {
				return;
			}
```